src/components/CatalogOverlay/CatalogOverlayPlotSettingsPanelComponent/CatalogOverlayPlotSettingsPanelComponent.tsx

```diff
@@ -820,15 +836,18 @@ export class CatalogOverlayPlotSettingsPanelComponent extends React.Component<Wi
     };
 
     private handleSelectedTabChanged = (newTabId: React.ReactText) => {
-        this.widgetStore.setSettingsTabId(Number.parseInt(newTabId.toString()));
+        this.widgetStore?.setSettingsTabId(Number.parseInt(newTabId.toString()));
     };
 
     private handleSelectedAxisTabChanged = (newTabId: React.ReactText) => {
-        this.widgetStore.setSizeAxisTab(Number.parseInt(newTabId.toString()));
+        this.widgetStore?.setSizeAxisTab(Number.parseInt(newTabId.toString()));
     };
 
     private getCatalogShape = (shape: CatalogOverlayShape) => {
         const widgetStore = this.widgetStore;
+        if (!widgetStore) {
+            return <Icon icon="circle" color={Colors.TURQUOISE3} />;
+        }
         let color = widgetStore.catalogColor;
```

Comment:
would make sense to just do `const color = widgetStore?.catalogColor ?? Colors.TURQUOISE3;`